src/stories/Button.stories.ts

```
@@ -0,0 +1,53 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { fn } from "@storybook/test";
+
+import { Button } from "./Button";
```

ESLint

[error] Missing file extension for "./Button"

(import/extensions)